index.js

```diff
@@ -2,113 +2,113 @@ const builder = require('botbuilder')
 let lib = new builder.Library('FormBuilder')
 
 lib.dialog('/', [
-  (session, args) => {
-    if (args !== null && typeof args !== 'undefined') {
-      if (!args.questions) {
-        return session.error('Missing questions array')
-      } else {
-        session.dialogData.questions = args.questions
-      }
-      session.dialogData.entities = args.entities ? args.entities : false
-      session.dialogData.index = args.index ? args.index : 0
-      session.dialogData.form = args.form ? args.form : {}
-      session.dialogData.reprompt = args.reprompt ? args.reprompt : false
-    } else {
-      return session.error('Arguments were not supplied or were undefined')
-    }
+    (session, args) => {
+        if (args !== null && typeof args !== 'undefined') {
+            if (!args.questions) {
+                return session.error('Missing questions array')
+            } else {
+                session.dialogData.questions = args.questions
+            }
+            session.dialogData.entities = args.entities ? args.entities : false
+            session.dialogData.index = args.index ? args.index : 0
+            session.dialogData.form = args.form ? args.form : {}
+            session.dialogData.reprompt = args.reprompt ? args.reprompt : false
+        } else {
+            return session.error('Arguments were not supplied or were undefined')
+        }
 
-    let index = session.dialogData.index
-    let questions = session.dialogData.questions
+        let index = session.dialogData.index
+        let questions = session.dialogData.questions
 
-    // Check if entities exists
-    if (session.dialogData.entities) {
-      // If the entities exists and it possesses the property for this question, send a confirm prompt
-      if (session.dialogData.entities.hasOwnProperty(questions[index].field)) {
-        var prompt = questions[session.dialogData.index].prompt
-        // Replace placeholder text with user data
-        prompt = prompt.replace('{' + questions[index].field + '}', session.dialogData.entities[questions[index].field])
-        builder.Prompts.confirm(session, prompt)
-      } else {
-        if (!session.dialogData.reprompt) {
-          builder.Prompts.text(session, questions[index].question)
+        // Check if entities exists
+        if (session.dialogData.entities) {
+            // If the entities exists and it possesses the property for this question, send a confirm prompt
+            if (session.dialogData.entities.hasOwnProperty(questions[index].field)) {
+                var prompt = questions[session.dialogData.index].prompt
+                // Replace placeholder text with user data
+                prompt = prompt.replace('{' + questions[index].field + '}', session.dialogData.entities[questions[index].field])
+                builder.Prompts.confirm(session, session.localizer.gettext(session.preferredLocale(), prompt))
```

Comment:
You're right. I wasn't sure what you were doing here at first but after looking closer I see you're replacing a placeholder variable with some user content. Pushed the change :)